test/k8sT/Services.go

```diff
@@ -1059,10 +1059,11 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 					})
 
 					SkipItIf(func() bool {
-						// Quarantine when running with the third node as it's
-						// flaky. See #12511.
-						return helpers.GetCurrentIntegration() != "" ||
-							(helpers.SkipQuarantined() && helpers.ExistNodeWithoutCilium())
+						// Quarantine under all circumstances, as it's flaky,
+						// especially when running with the third node. See
+						// https://github.com/cilium/cilium/issues/12511 and
+						// https://github.com/cilium/cilium/issues/12690.
+						return helpers.GetCurrentIntegration() != "" || helpers.SkipQuarantined()
```

Comment:
This test case when running without the third node (i.e., the node which doesn't run cilium-agent) is useless, as it is intended to test Cilium's N/S loadbalancing for the particular configuration. The latter can be tested only by the third node.

Reply:
Why is it run when there is no third node then? Was that a mistake?

Reply:
I think the previous assumption was that KPR=strict was only when there existed the third node which is no longer true.

Reply:
Ok. So we should now just skip it if there is no node without Cilium, regardless of whether it's quarantined?
```suggestion
						return helpers.GetCurrentIntegration() != "" || helpers.DoesNotExistNodeWithoutCilium() || helpers.SkipQuarantined()
```

Reply:
:+1:

Reply:
Thanks, PR updated.